CONTRIBUTING.md

```diff
@@ -106,7 +106,9 @@ Testing can be accomplished on systems capable of hosting supported container ru
 
 ### Using GCP
 
-Convenience functions (change the Default name, the first variable):
+**If you are a Replicated team member then:** ensure that you follow up the doc [Getting Started with your own CodeServer][code-server].
```

Comment:
```suggestion
**If you are a Replicated team member then:** ensure that you have followed [Getting Started with your own CodeServer](https://github.com/replicatedhq/codeserver/blob/main/docs/first-time.md).
```

Reply:
Use alias make easier we keep the docs maintained since all links would be at the bottom

Reply:
However, it is not a big deal eaither.